package.json

```diff
@@ -23,6 +23,7 @@
     "@typescript-eslint/parser": "^2.0.0",
     "cross-env": "^5.2.1",
     "eslint": "^6.2.2",
+    "eslint-import-resolver-ts": "^0.4.2",
```

Comment:
This helps resolve the type imports like in https://github.com/frontity/frontity/pull/262/files#diff-d1b3474f889aadcfb57de4d0c9c26b08R2.

Reply:
Good catch. I saw the package as the solution to the type import problem and didn't notice it ws a fork. The main repo should be used.